tests/unit/test_download.py

```diff
@@ -350,6 +405,31 @@ def tearDown(self):
         self.files.remove_all()
 
 
+class TestIORenameFileTask(BaseIOTaskTest):
```

Comment:
I think we should call this TestIOStreamingWriteTask. Probably forgot to change on copy and paste.